frontend/src/hooks/queryBuilder/useFetchKeysAndValues.ts

```diff
@@ -151,10 +163,19 @@ export const useFetchKeysAndValues = (
 		}
 	}, [data?.payload?.attributeKeys, status]);
 
+	useEffect(() => {
```

Comment:
can we make use of onSuccess here

Reply:
sure, I'll move to the useQuery for this request

Reply:
updated